include/msgpack/cpp_config.hpp

```diff
@@ -24,7 +24,10 @@
   // If MSVC would support C++11 completely,
   // then 'defined(_MSC_VER)' would replace with
   // '_MSC_VER < XXXX'
-# if (__cplusplus < 201103L) || defined(_MSC_VER)
+# if (__cplusplus < 201103L) && !defined(_MSC_VER)
+#  define MSGPACK_USE_CPP03
+# endif
+# if defined(_MSC_VER) && (_MSC_VER < 1900)
```

Comment:
I don't think 2015 has an 03 mode, but this is only temporary anyway, I expect VS to come into compliance with an update.